deploy-board/deploy_board/templates/hosts/host_details.html

```diff
@@ -41,18 +41,24 @@
 {% endif %}
 
 {% if agent_wrappers %}
-    <div class="panel panel-default">
-    {% include "panel_heading.tmpl" with panel_title="Agent Details" panel_body_id="agentDetailsId" direction="down" copy_host_name_button=True %}
+<div class="panel panel-default">
+    {% include "panel_heading.tmpl" with panel_title="Agent Details" panel_body_id="agentDetailsId" direction="down"
+    copy_host_name_button=True %}
     <div id="agentDetailsId" class="collapse in panel-body">
-
-    {% for agent_wrapper in agent_wrappers %}
+        {% for key, value in host_details.items %}
```

Comment:
This code works now, but I need help here to avoid extra for loop to get ip value from host_details map.

Reply:
Or any idea for pass in host ip variable in a better way?

Reply:
the key 'phobos link' might not exists for non pinterest(for example), we still want to show the agent_wrapper for loop. so it is better to use custom filter see the end of https://stackoverflow.com/questions/2970244/django-templates-value-of-dictionary-key-with-a-space-in-it
add the custom filter in deploy-board/deploy_board/webapp/templatetags/utils.py

Reply:
Thanks for the info, updated.